chart/values.yaml

```
@@ -29,13 +29,19 @@ postgres:
 
 sso:
   enabled: true
+  # Options: "gitlab", "openid_connect"
+  protocol: "gitlab"
   secretName: mattermost-sso
 
   # These should typically be disabled if SSO is enabled
   enable_sign_up_with_email: false
   enable_sign_in_with_email: false
   enable_sign_in_with_username: false
 
+  # Config for OIDC/SAML
+  login_button_text: "Login with SSO"
+  login_button_color: "#145DBF"
```

This should default to the Defense Unicorns blue as defined in our brand guide.

```suggestion
  login_button_color: "#144A8F"
```